mteb/models/nomic_models.py

```diff
@@ -28,6 +32,17 @@ def __init__(
         **kwargs: Any,
     ):
         self.model_name = model_name
+        if (
+            model_name == "nomic-ai/modernbert-embed-base"
+            and CURRENT_TRANSFORMERS_VERSION < MODERN_BERT_TRANSFORMERS_MIN_VERSION
+        ):
+            min_version_str = ".".join(
+                [str(v) for v in MODERN_BERT_TRANSFORMERS_MIN_VERSION]
+            )
+            raise RuntimeError(
+                f"Current transformers version is {transformers.__version__} is lower than the required version"
+                f" {min_version_str}"
+            )
```

Comment:
Could use packaging.verison (combined with above):
```suggestion
        if (
            model_name == "nomic-ai/modernbert-embed-base"
            and version.parse(transformers.__version__) < version.parse(MODERN_BERT_TRANSFORMERS_MIN_VERSION):
            RuntimeError(
                f"Current transformers version is {transformers.__version__} is lower than the required version"
                f" {MODERN_BERT_TRANSFORMERS_MIN_VERSION}"
            )
```

Reply:
very minor but I think you can further do:
```
from packaging.version import Version

Version(transformers.__version__) < Version(MODERN_BERT_TRANSFORMERS_MIN_VERSION)
```
Which simply removed the parse statement.